qstnrs/qstnrs/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^qstnrs/', include('questionnaire.urls')),
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
Qstnrs is the django project and Questionnaire is the app, so the django admin urls already live in the django project. Isn't this correct?